src/server/routes/sign.rs

```diff
@@ -55,7 +55,7 @@ struct SignedBody {
     validator_signature: String,
 }
 
-pub async fn sign_route<Config>(
+pub async fn sign_route<'a, Config>(
```

Comment:
What requires this added lifetime parameter? I cannot see any change requiring it.